wcivf/settings/base.py

```diff
@@ -121,7 +121,7 @@
         "ENGINE": "django.db.backends.postgresql_psycopg2",
         "NAME": "wcivf",
         "USER": os.environ.get("DB_USER", "wcivf"),
-        "PASSWORD": os.environ.get("DB_PASSWORD", ""),
+        "PASSWORD": os.environ.get("DB_PASSWORD", "wcivf"),
```

Comment:
We might want this in a Lambda settings file that we can import from - this will mean local installs will need a password too